install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -386,6 +386,15 @@ data:
   # https://github.com/cilium/hubble/blob/master/Documentation/metrics.md
   hubble-metrics: {{ .Values.global.hubble.metrics | join " " | quote }}
 {{- end }}
+{{- range .Values.global.hubble.listenAddresses }}
+        - --listen-client-urls={{ . }}
+{{- end }}
+{{- if and .Values.hubble-ui.enabled (not (has ":50051" .Values.global.hubble.listenAddresses)) }}
+        - --listen-client-urls=0.0.0.0:50051
+{{- if not .Values.global.hubble.listenAddresses }}
```

Comment:
This is not needed anymore. Embedded Hubble works slightly different than old standalone Hubble and has the `hubble-socket-path` configmap option for this. So you can just remove this if statement here.